Remove TF dependency from Strawberry Fields (#88)
XanaduAI/strawberryfields: doc/installing.rst

* `NumPy <http://numpy.org/>`_  >=1.13.3
* `SciPy <http://scipy.org/>`_  >=1.0.0
* `NetworkX <http://networkx.github.io/>`_ >=2.0
* `Blackbird <https://quantum-blackbird.readthedocs.io>`_ >=0.2.0

If you currently do not have Python 3 installed, we recommend `Anaconda for Python 3 <https://www.anaconda.com/download/>`_, a distributed version of Python packaged for scientific computation.

Installation
============

Installation of Strawberry Fields, as well as all required Python packages mentioned above, can be installed via ``pip``:

.. code-block:: console

    pip install strawberryfields

TensorFlow support
==================

> To use Strawberry Fields with TensorFlow, version 1.3 of

Reviewer: A lot of this is duplicated in the TFBackend file itself. Can we not get sphinx to read that in here?

Author: Not that I am aware. If the file was a `rst` file we could, using `.. include::`, but it wouldn't work with autodoc.

TensorFlow is required. This can be installed alongside Strawberry Fields
as follows:

.. code-block:: console

    pip install strawberryfields tensorflow==1.3

Or, to install Strawberry Fields and TensorFlow with GPU and CUDA support:

.. code-block:: console

    pip install strawberryfields tensorflow-gpu==1.3

> Note that TensorFlow version 1.3 is only support on Python version

Reviewer: ```suggestion
Note that TensorFlow version 1.3 is only supported on Python versions
```

less than 3.7. You can use the following command to check your
Python version:

.. code-block:: console

    python --version

> If the above prints out 3.7, you will need to install Python 3.6.

Reviewer: ```suggestion
If the above prints out 3.7, and you still want to use TensorFlow, you will need to install Python 3.6.
```

> The recommended method is to install `Miniconda3 <https://docs.conda.io/en/latest/miniconda.html>`_.

Reviewer: elsewhere we recommend and link to Anaconda. Should we keep same recommendation throughout?

Author: Oh, I missed that. Should I update it to miniconda everywhere, or stick with anaconda?

Author: I've updated to miniconda everywhere

Once installed, you can then create a Python 3.6 Conda environment:

.. code-block:: console

>     conda create --name new_env python=3.6

Reviewer: ```suggestion
    conda create --name sf_tensorflow_env python=3.6
```

>     conda activate new_env

Reviewer: ```suggestion
    conda activate sf_tensorflow_env
```
